app/models/article.rb

```diff
@@ -1,4 +1,6 @@
 class Article < ApplicationRecord
+
+    belongs_to :category
     validates :header, presence: true
```

Comment:
We should create a chore to refactor these to validates_presence_of

Reply:
In a refactoring chore. Not now IMO

Reply:
agreed - hence "We should create a chore to refactor these to validates_presence_of" :)